_sass/components/_identifier.scss

```diff
@@ -0,0 +1,134 @@
+.contact-bar {
+  padding-bottom: 0.5rem;
+  padding-top: 0.5rem;
+}
+
+.usa-identifier {
+  background-color: $theme-identifier-background-color;
+  color: $theme-identifier-color;
+  font-family: $font-sans;
+  font-weight: 400;
+  font-size: 1.5rem;
+  padding-top: 1rem;
+  padding-bottom: 3rem;
+
+  a {
+    color: $theme-identifier-color;
+  }
+
+  ul, ol {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+}
+
+.usa-identifier__required-links-list a {
+  font-size: 1.5rem;
+}
+
+a.usa-identifier__required-link {
+  color: $theme-identifier-color;
+
+  &:hover {
+   color: $theme-identifier-text-color-secondary;
+  }
+}
+
+.usa-identifier__container {
+  @media #{$breakpoint-lg} {
+    display: flex;
+    align-items: center;
+  }
+}
+
+.usa-identifier__logos {
+  padding-top: 1rem;
+  padding-right: 2rem;
+}
+
+.usa-identifier__logo {
+  display: block;
+  width: 5rem;
+}
+
+.usa-identifier__identity-disclaimer {
+  font-size: 1.6rem;
+}
+
+.usa-identifier__identity-disclaimer a,
+.usa-link--external,
+a.usa-identifier__identity-disclaimer-link {
+  font-weight: 700;
+}
+
+.usa-identifier__identity-disclaimer-link a,
+.usa-identifier__identity-disclaimer a:visited {
+  color: $theme-identifier-text-color-secondary;
+}
+
+a.usa-identifier__identity-disclaimer-link:hover,
+.identifier__identity-disclaimer a:hover {
+  color: $theme-identifier-link-color;
+}
+
+.usa-identifier__identity {
+  padding-top: 1rem;
+
+  @media #{$breakpoint-lg} {
+    padding-top: 0;
+  }
+
+  p {
+    line-height: 1;
+    margin: .5rem 0 0 0;
+    padding: 0;
+  }
+}
+
+.usa-identifier__identity-domain {
+  color: $theme-identifier-color;
+  font-family: $font-sans;
+  font-weight: 400;
+  font-size: 1.6rem;
+}
+
+.usa-identifier__identity-disclaimer {
+  color: $theme-identifier-text-color;
+  font-family: $font-sans;
```

Comment:
Same with this font-family declaration, let's delete.

Reply:
This is removed